lib/visitor.rb

```diff
@@ -0,0 +1,26 @@
+class Visitor
+    attr_reader :name, :height, :spending_money, :preferences
+
+    def initialize(name, height, spending_money)
+        @name = name
+        @height = height
+        @spending_money = spending_money.delete("$").to_i
+        @preferences = []
+    end
+
+    def add_preference(preference)
+        preferences << preference
+    end
+
+    def tall_enough?(requirement)
+        if @height >= requirement
+            return true
+        else
+            return false
+        end
+    end
+
+    def take_money(amount)
+        @spending_money -= amount
+    end
```

Comment:
Nice job writing a method to help with changing this attribute.

Reply:
You will want to make sure you write a test for this method though, to ensure it's working how you want it to.

Reply:
I couldn't get this to run anyways